txselector/errors.go

```diff
@@ -65,4 +65,11 @@ const (
 	ErrNoAvailableSlotsCode int = 17
 	// ErrNoAvailableSlotsType error type
 	ErrNoAvailableSlotsType string = "ErrNoAvailableSlots"
+
+	// ErrInvalidAtomicGroup error message returned if an atomic group is malformed
+	ErrInvalidAtomicGroup = "Tx not selected because it belongs to an atomic group with missing transactions or bad requested tranaction"
```

Comment:
typo in the last word `tranaction`

in order to avoid mistakes like this, if you are a `VS Code` user, this extension can help https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker